views/widget.html

```diff
@@ -0,0 +1,11 @@
+{% set widget = data.widget %}
+{% if widget.linkType == 'page' %}
+  {% set path = widget._linkPage._url %}
+{% elif widget.linkType == 'custom' %}
+  {% set path = widget.linkUrl %}
+{% endif %}
+{% if data.options.classes.length > 0 %}
+  {% set classes = true %}
+  {% set class = data.options.selectors | join(' ') %}
```

Comment:
`data.options.selectors` = `data.options.classes`

Reply:
Bug - you fixed it to .classes in one place but it's still .selectors in another so it won't work.